test/fixtures/user/user.js

```diff
@@ -35,7 +35,10 @@ module.exports = () => {
       linkedin_id: 'nikhil-bhandarkar',
       twitter_id: 'whatifi',
       phone: '1234567891',
-      email: 'abc1@gmail.com'
+      email: 'abc1@gmail.com',
```

Comment:
Can we create another user object here?
Adding restricted to this user will affect the other non-GET requests where this is being used.

Reply:
@ankurnarkhede Good point. I will just add one more user.